metadata.json

```diff
@@ -283,14 +305,18 @@
     "browser.devtools.panels.create",
     "browser.devtools.panels.elements",
     "browser.downloads.download",
+    "browser.offscreen.closeDocument",
     "browser.offscreen.createDocument",
+    "browser.runtime)",
```

Comment:
🤔 Might need to trim: https://github.com/wxt-dev/examples/pull/5

Reply:
I would rather update the code so the actual API you use inside `browser.runtime.xyz` shows up.

Reply:
Now that you mention it, you're right! 😄